arangodb-net-standard.Test/CollectionApi/CollectionApiClientTest.cs

```diff
@@ -261,5 +261,63 @@ public async Task GetCollectionPropertiesAsync_ShouldSucceed()
             Assert.Equal(3, response.Status);
             Assert.Equal(2, response.Type);
         }
+        
+        [Fact]
+        public async Task RenameCollectionAsync_ShouldSucceed()
+        {
+            await _adb.Collection.PostCollectionAsync(
+                    new PostCollectionBody
+                    {
+                        Name = "TempCollection"
```

Comment:
What about using the name of the test method as a test collection? e.g. `string initialClx = nameof(RenameCollectionAsync_ShouldSucceed)` and `string renamedClx = initialClx + "_Renamed"`